### PR TITLE
[Documentation] Chat Version Update

### PR DESCRIPTION
**Pull request by jpeng-ms**
## Purpose
Updated version to 1.0.0-beta.2.

Additional PR would need to update:
- release date
- podspec tag

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## Pull Request Type
What kind of change does this Pull Request introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation content changes
- [x] Other... Please describe:

## Pull request checklist

This PR has considered:
- [ ] Color Theming
- [ ] iOS supported versions (all major versions greater than or equal current target deployment version)
- [ ] VoiceOver and Keyboard Accessibility
- [ ] Internationalization and Right to Left layouts
- [ ] Size classes and window sizes (iPhone vs iPad, notched devices, multitasking, different window sizes, etc)
- [ ] View Data Injection
- [ ] Demo App UI/UX update

## How to Test

* Open...
* Click on...

## What to Check

| Before                                       | After                                      |
|----------------------------------------------|--------------------------------------------|
| Screenshot or description before this change | Screenshot or description with this change |

### CODE REVIEW

**Review by palatter — DISMISSED** [review]
Looks good.

File: AzureCommunicationUI/sdk/AzureCommunicationUIChat/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Release History
 
+## 1.0.0-beta.2 (TBA)
+### Bugs Fixed
+- Addressed the issue where LocalizationProvider not being marked as internal.
+- Addressed the issue where README's banner image not shown up on API doc.
```
Comment (jimchou-dev) [inline AzureCommunicationUI/sdk/AzureCommunicationUIChat/CHANGELOG.md:6]: I think this is just changing documentation, don't need to be in bugfix

**Review by jimchou-dev — DISMISSED** [review]
LGTM

File: AzureCommunicationUI/sdk/AzureCommunicationUIChat/CHANGELOG.md (hunk 1 above)
Comment (jpeng-ms) [reply AzureCommunicationUI/sdk/AzureCommunicationUIChat/CHANGELOG.md:6]: as discussed with Jim offline, will remove the image changes from the changelog and update the section to "other changes"

**Review by jimchou-dev — APPROVED** [review]

**Review by palatter — APPROVED** [review]

**Pull request merged**